Block/Js.php

```diff
@@ -470,4 +466,30 @@ public function minifyJs($js)
             return $js;
         }
     }
+
+    /**
+     * Return true if we are on cart page or checkout page
+     */
+    public function isOnPageFromWhiteList() {
+        $currentPage = $this->getRequest()->getFullActionName();
+        return in_array($currentPage, $this->getPageWhitelist());
+    }
+
+    /**
+     * Return true if bolt on minicart is enabled
+     */
+    public function isMinicartPage() {
+        return $this->configHelper->getMinicartSupport();
+    }
+
+    /**
+     * Return true if we are on product page, and bolt on product page is enabled
+     */
+    public function isProductPage() {
```

Comment:
I think this name is a little vague - it doesn't _only_ check if it's a product page. Maybe something like `isBoltProductPage`?

Reply:
Fixed